tests/integration/buildcmd/test_build_cmd.py

```diff
@@ -736,6 +736,29 @@ def test_build_single_layer(self, runtime, use_container, layer_identifier):
             "python",
         )
 
+    @parameterized.expand(
+        [("makefile", False, "LayerWithMakefile"), ("makefile", "use_container", "LayerWithMakefile")]
+    )
+    def test_build_layer_with_makefile(self, runtime, use_container, layer_identifier):
```

Comment:
nit: instead of `runtime`, it can be `build_method`

Reply:
Sure.